Userland/Utilities/gunzip.cpp

```diff
@@ -41,10 +41,13 @@ ErrorOr<int> serenity_main(Main::Arguments args)
         keep_input_files = true;
```

Comment:
`assignation` does not mean what you think it means :) the correct word here would be assignment